### PR TITLE
Issue 6 - Include a list of related items

### LINKED ISSUE #6: Include a list of related items
The `amp-list` component fetches dynamic content from a CORS JSON endpoint and renders it using a supplied template. This is good for embedding a dynamic list of related articles, but is not exactly what we need in the case of Plone.

The list of related items can be displayed as a simple list of items:

``` html
  <ul>
    <li>
      <a href="url">title</a>
    </li>
  </ul>
```

### PR DESCRIPTION
**Pull request by rodfersou**
closes #6

### CODE REVIEW

File: src/collective/behavior/amp/browser/view.pt [hunk 1]
```diff
@@ -54,6 +54,23 @@
     <p tal:condition="context/text/output|nothing"
         tal:replace="structure context/text/output">Body text</p>
 
+    <div class="amp-related">
+      <span i18n:translate="label_related_items">Related content</span>
+      <ul tal:define="site_properties context/portal_properties/site_properties;
```
Comment (hvelarde) [inline src/collective/behavior/amp/browser/view.pt]: @rodfersou the portal_properties tool has been removed in Plone 5, so this is not going to work; please find out how to make this compatible with both versions.

File: src/collective/behavior/amp/tests/test_view.py [hunk 2]
```diff
@@ -108,6 +111,21 @@ def test_amp_analytics(self):
         self.assertEqual(
             amp.find('*//amp-analytics/script').text, u'"foo": "bar"')
 
+    def test_related_items(self):
```
Comment (hvelarde) [inline src/collective/behavior/amp/tests/test_view.py]: @rodfersou I would prefer to test against the HTML code rendered as in the other features.

File: src/collective/behavior/amp/browser/view.pt (hunk 1 above)
Comment (rodfersou) [reply src/collective/behavior/amp/browser/view.pt]: Done

File: src/collective/behavior/amp/browser/__init__.py [hunk 3]
```diff
@@ -166,6 +176,51 @@ def text(self):
...
 
+    def use_view_action(self):
+        if IS_PLONE_5:
+            use_view_action = api.portal.get_registry_record(
+                'plone.types_use_view_action_in_listings'
+            )
+        else:
+            site_properties = api.portal.get_tool('portal_properties').site_properties
+            use_view_action = site_properties.typesUseViewActionInListings
+        if not use_view_action:
```
Comment (hvelarde) [inline src/collective/behavior/amp/browser/__init__.py]: @rodfersou in which circumstances this will be executed?

**Pull request merged**